plugins/builds/update.js

```diff
@@ -7,6 +7,32 @@ const schema = require('screwdriver-data-schema');
 const { EXTERNAL_TRIGGER } = schema.config.regex;
 const idSchema = joi.reach(schema.models.job.base, 'id');
 
+/**
+ * Determine if this build is FIXED build or not.
+ * @method isFixedBuild
+ * @param  build
+ * @param  jobFactory
+ */
+function isFixedBuild(build, jobFactory) {
+    return jobFactory.get(build.jobId).then(job =>
+        job.getLatestBuild({ status: 'FAILURE' }).then(failureBuild =>
+            job.getLatestBuild({ status: 'SUCCESS' }).then(successBuild => {
+                if (!failureBuild) {
+                    return false;
+                }
+                if (failureBuild && !successBuild) {
+                    return true;
```

Comment:
Is there no `SUCCESS` builds? It seems that it should be true?

Reply:
I thought it was necessary if the build has never been successful.
For example, the following cases

| Build ID | Job ID | status | 
|:---:|:---:|:---:|
| 1 | 111 | FAILURE |
| 2 | 111 | FAILURE |
| 3 | 111 | SUCCESS ← At this time, No SUCCESS in the past.|